### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by liebeskind**
Topia now supports screenshare and is peer-to-peer

### CODE REVIEW

**Comment by billmei** [comment]
Thanks! Is Topia is end-to-end encrypted? I couldn't find this information on the website. If not, please remove "encrypted" from the description.

**Comment by liebeskind** [comment]
> 

Bill, our core (standard) product is peer-to-peer webRTC, which is end-to-end encrypted video chat. Watch [this video](https://www.youtube.com/watch?v=a7OEKjiLu7Q&t) for a much more in-depth explanation.

**Comment by billmei** [comment]
Since webRTC is a common technology used by most other video conferencing applications, I hesitate to accept "encrypted" as part of the description because it could imply a level of security your product has that others do not. It would be like labelling your product as "[asbestos-free](https://xkcd.com/641/)", which while technically true, feels somewhat misleading. My understanding (please correct me if this is wrong) is also that using a TURN server (in what you call "performance mode") with webRTC is not considered e2e encrypted. If you'd like to highlight Topia's security details, you're welcome to do so in other ways, for example the fact that you participate in a vulnerability disclosure program, but otherwise I will accept your pull request without the word "encrypted".

Thanks :)

**Comment by liebeskind** [comment]
@billmei that's fine with me!

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ These apps can be run directly in the browser.
 - [Spot](https://www.spot.xyz): Fully customizable virtual office space
 - [Sprout](https://sprout.place/): Virtual whiteboard with video calling
 - [Teamflow](https://teamflowhq.com): Virtual office space for enterprise
-- [Topia](https://topia.io): Proximity video chat in a customizable virtual world (does not support screensharing)
+- [Topia](https://topia.io): Peer-to-Peer encrypted proximity video chat in customizable virtual worlds
```
Comment (billmei) [inline README.md]:
```suggestion
- [Topia](https://topia.io): Peer-to-Peer proximity video chat in customizable virtual worlds
```

**Pull request merged**